tests/Unit/ValueObject/DateValueObjectTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Unit\Patterns;
+
+use Drupal\oe_theme\ValueObject\DateValueObject;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Test date value object.
+ */
+class DateValueObjectTest extends UnitTestCase {
+
+  /**
+   * Test constructing a date value object from an array.
+   */
+  public function testFromArray() {
```

Comment:
Do we need to keep this test if we already have the DateBlockPatternRenderingTest.php? If so, we still need a test for the fromTimestamp method here